### PR TITLE
ENS integration in wallet transfer page

### PR DESCRIPTION
**Pull request by rathishubham7**

### CODE REVIEW

File: app/src/config.js [hunk 1]
```diff
@@ -50,6 +50,6 @@ export default {
   REDDIT_CLIENT_ID: 'dcQJYPaG481XyQ',
   DISCORD_CLIENT_ID: '630308572013527060',
   redirect_uri: redirect_uri,
-  // api: 'http://localhost:2020'
-  api: 'https://api.tor.us'
+  api: 'http://localhost:2020'
+  // api: 'https://api.tor.us'
```
Comment (chaitanyapotti) [inline app/src/config.js]: change this back pls

File: app/src/store/store.js [hunk 2]
```diff
@@ -230,13 +230,19 @@ VuexStore.subscribe((mutation, state) => {
...
           created_at: new Date(txMeta.time),
           from: toChecksumAddress(txMeta.txParams.from),
           to: toChecksumAddress(txMeta.txParams.to),
           total_amount: totalAmount,
+          gas: txMeta.txParams.gas,
+          gasPrice: txMeta.txParams.gasPrice,
+          nonce: txMeta.txParams.nonce,
+          type: txMeta.contractParams && txMeta.contractParams.erc20 ? 'erc20' : txMeta.contractParams.erc721 ? 'erc721' : 'eth',
+          type_name: txMeta.contractParams && txMeta.contractParams.name ? txMeta.contractParams.name : '',
+          type_image_link: txMeta.contractParams && txMeta.contractParams.logo ? txMeta.contractParams.logo : '',
```
Comment (chaitanyapotti) [inline app/src/store/store.js]: where is the logo coming from?

File: app/src/config.js (hunk 1 above)
Comment (rathishubham7) [reply app/src/config.js]: yes.

File: app/src/store/store.js (hunk 2 above)
Comment (rathishubham7) [reply app/src/store/store.js]: transactionController.js

**Comment by rathishubham7** [comment]
1. ENS integration
2. Changes to walletHistory page.

**Pull request merged**